Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -1691,7 +1691,6 @@ internal override HttpRequestMessageBuilder BuildRequest(HttpRequestMessageBuild
                 new XElement("ETag", ETags[i])));
         var completeMultipartUploadXml = new XElement("CompleteMultipartUpload", parts);
         var bodyString = completeMultipartUploadXml.ToString();
-        var body = Encoding.UTF8.GetBytes(bodyString);
```

Comment:
unrelated to `HttpRequestMessageBuilder`, but this was not being used and just converting the string twice.